cl/corpus_importer/tests.py

```diff
@@ -5,6 +5,7 @@
 from unittest.mock import patch
 
 import pytest
+from courts_db import find_court
```

Comment:
Is this used?

Reply:
I forgot to optimize the imports, one sec